std_msgs/Header header
geometry_msgs/PoseArray poseArray 
nav_msgs/Path path
geometry_msgs/Twist[] twists
